django/db/models/functions/text.py

```diff
@@ -4,6 +4,11 @@
 from django.db.models.functions import Coalesce
 from django.db.models.lookups import Transform
 
+try:
+    from psycopg3.types.numeric import Int4
```

Comment:
Please import this from `psycopg3.types`. I haven't made documentation yet, I'm making my mind just in these days, but I think the `types` package will be the facade to access all the objects (flat is better than nested, and internal modules may become an implementation detail).

note: if that's the way you found it elsewhere, please change it: I really started importing in the `types` packages just a few days ago :)

Reply:
Fixed.